.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature Request
+about: Ask for missing features or improvements
+
+---
+
+#### Description
+<!-- Please describe your use case, why you need this feature and why this
+feature is important for RIOT. -->
+
+### Useful links
+<!-- Please include links to any documentation that you think is useful -->
```

Comment:
Full stop after "useful"

Reply:
Seems nit-picky, but these things do have an effect on how professional we seem to contributors...

Reply:
After that, ACK from me, please squash and rebase

Reply:
Fixed, squashed and rebased.